ibis/backends/tests/test_map.py

```diff
@@ -26,6 +26,29 @@
 ]
 
 
+@pytest.mark.notyet("clickhouse", reason="nested types can't be NULL")
+@pytest.mark.broken(["pandas", "dask"], reason="TypeError: iteration over a 0-d array")
+@pytest.mark.notimpl(
+    ["risingwave"],
+    raises=PsycoPg2InternalError,
+    reason="function hstore(character varying[], character varying[]) does not exist",
+)
+@pytest.mark.parametrize(
+    "kv",
+    [
+        param((["a", "b"], None), id="null_values"),
+        param((None, ["c", "d"]), id="null_keys"),
+        param((None, None), id="null_both"),
+    ],
+)
+def test_map_nulls(con, kv):
+    k, v = kv
```

Comment:
```suggestion
    ("k", "v"),
    [
        param(["a", "b"], None, id="null_values"),
        param(None, ["c", "d"], id="null_keys"),
        param(None, None, id="null_both"),
    ],
)
def test_map_nulls(con, k, v):
```

Let pytest do the work for you :)

Reply:
thanks, done.